pkg/controller/node/node_controller.go

```diff
@@ -721,6 +721,9 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 	}
 
 	if pool.Spec.Paused {
+		if mcfgv1.IsMachineConfigPoolConditionTrue(pool.Status.Conditions, mcfgv1.MachineConfigPoolUpdating) {
+			glog.Infof("Pool %s is paused and will not update.", pool.Name)
+		}
 		return ctrl.syncStatusOnly(pool)
```

Comment:
on a second thought - while this log looks good, it's still gonna be pretty hidden within our logs which is again not giving any useful information to the admin

one possible solution would be to enhance `syncStatusOnly` with logic to show a message in the Updating condition (or somewhere else) saying it's Paused (we ultimately need something at the MCP <-> MCO objects level to be admin-visible)

Reply:
Makes sense I think we can do this via a follow-on pr.